lib/koala/api.rb

```diff
@@ -61,7 +61,7 @@ def api(path, args = {}, verb = "get", options = {}, &error_checking_block)
         end
 
         # Translate any arrays in the params into comma-separated strings
-        args = sanitize_request_parameters(args)
+        args = sanitize_request_parameters(args) if !options[:format] || (options[:format] && options[:format].to_s.downcase != "json")
```

Comment:
I think we can make it simpler by just saying `if options[:format] != :json` -- this is an internal option, so it's okay to define a particular way it should be passed in. (I'll do that as I merge it in.)